api/src/pcapi/scripts/offerer/synchronize_venues_banners_with_google_places.py

```diff
@@ -0,0 +1,190 @@
+    for photo in photos:
+        if is_the_owner(owner_name, photo):
+            return photo
+    return None
+
+
+storage_folder = settings.THUMBS_FOLDER_NAME + "/google_places"
+
+
+def save_photo_to_gcp(venue_id: int, photo: PlacePhoto, prefix: str) -> str:
+    photos_result = gmaps.places_photo(
+        photo_reference=photo.photo_reference,
+        max_width=photo.width,
+    )
+    object_id = f"google_places_{prefix}{venue_id}.jpeg"
+    data = bytes()
+    for chunk in photos_result:
+        if chunk:
+            data += chunk
+    store_public_object(
+        storage_folder,
+        object_id,
+        image_conversion.standardize_image(
+            content=data, ratio=image_conversion.ImageRatio.LANDSCAPE, crop_params=image_conversion.CropParams  # type: ignore [arg-type]
+        ),
+        "image/jpeg",
+    )
+    return settings.OBJECT_STORAGE_URL + "/" + storage_folder + "/" + object_id
+
+
+@blueprint.cli.command("synchronize_venues_banners_with_google_places")
+@click.argument("begin", type=int, required=True)
+@click.argument("end", type=int, required=True)
+@click.argument("limit", type=int, required=False, default=None)
+def synchronize_venues_banners_with_google_places(begin: int, end: int, limit: int | None = None) -> None:
+    venues_without_photos = get_venues_without_photo(begin, end, limit)
+    print("the number of venues is:", len(venues_without_photos))
+    logger.info(
+        "[gmaps_banner_synchro] synchronize_venues_banners_with_google_places %s %s %s: the number of venues tried: %s",
+        begin,
+        end,
+        limit,
+        len(venues_without_photos),
+    )
+    number_of_places_found = 0
+    number_places_with_at_least_one_photo = 0
+    number_of_places_with_owner_photo = 0
+    for venue in venues_without_photos:
+        if venue.googlePlacesInfo:
+            number_of_places_found += 1
+        else:
+            place_id = get_place_id(venue.name, venue.address)
+            if not place_id:
+                continue
+            venue.googlePlacesInfo = offerers_models.GooglePlacesInfo(
+                placeId=place_id,
+            )
+
+        number_of_places_found += 1
+        place_details = get_place_photos_and_owner(venue.googlePlacesInfo.placeId)
+        if not (place_details and place_details.photos):
+            continue
+        number_places_with_at_least_one_photo += 1
+
+        owner_photo = get_owner_photo(place_details.photos, place_details.name)
+        if owner_photo:
+            photo = owner_photo
+            photo_prefix = "owner"
+            number_of_places_with_owner_photo += 1
+        else:
+            photo = place_details.photos[0]
+            photo_prefix = ""
+        try:
+            venue.googlePlacesInfo.bannerUrl = save_photo_to_gcp(venue.id, photo, photo_prefix)
+            venue.googlePlacesInfo.bannerMeta = photo.model_dump()
+        except image_conversion.ImageRatioError as e:
+            logger.error(
```

Comment:
[détail] `logger.exception()` permet de transmettre à Sentry le traceback. Ça peut être assez pratique pour certains cas d'erreur, où l'on est content de voir les détails de chaque frame.

```suggestion
            logger.exception(
```